tests/converters/downcast.js

```diff
@@ -26,7 +26,10 @@ function paragraphInTableCell() {
 
 				if ( viewElement && viewElement.name === 'span' ) {
 					conversionApi.mapper.unbindModelElement( tableCell );
+
+					conversionApi.writer.removeStyle( 'display', viewElement );
```

Comment:
I found such a function defined in tests quite odd. In fact, the whole _utils/utils.js file contains so much feature code that I'm afraid that many tests here are actually testing that code, not how the real table feature work in real life. 

Wouldn't it be possible to make the tests in `downcast.js` more realistic?

Reply:
cc @jodator

Reply:
Yes, I don't like this either after 2 years. Previously it was done probably to isolate conversion from other editing test. IIRC this might be done as other, less schema intensive, features were tested this way (by defining schema by hand) for conversion tests.

At this stage I'm confident that whole utils should be treated with :fire:.

To make those tests more realistic we should always use `TableEditing` for those tests.

Reply:
> To make those tests more realistic we should always use TableEditing for those tests.

👍 Let's keep that in mind for the future.